src/Tizen.Uix.Tts/Tizen.Uix.Tts/SynthesizedPcmEventArgs.cs

```diff
@@ -0,0 +1,85 @@
+﻿/*
+* Copyright (c) 2023 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+
+namespace Tizen.Uix.Tts
+{
+    /// <summary>
+    /// This class holds information related to the TTS SynthesizedPcm event.
+    /// </summary>
+    /// <since_tizen> 11 </since_tizen>
+    public class SynthesizedPcmEventArgs
+    {
+        internal SynthesizedPcmEventArgs(int utteranceId, SynthesizedPcmEvent synthesizedPcmEvent, byte[] data, AudioType audioType, int sampleRate)
+        {
+            UtteranceId = utteranceId;
+            SynthesizedPcmEvent = synthesizedPcmEvent;
+            Data = data;
+            AudioType = audioType;
+            SampleRate = sampleRate;
+        }
+
+        /// <summary>
+        /// The utterance ID.
+        /// </summary>
+        /// <since_tizen> 11 </since_tizen>
+        public int UtteranceId
+        {
+            get;
+            internal set;
+        }
+
+        /// <summary>
+        /// The event of the PCM data.
+        /// </summary>
+        /// <since_tizen> 11 </since_tizen>
+        public SynthesizedPcmEvent SynthesizedPcmEvent
```

Comment:
I like the name changes for other properties - removing prefix that is already contained in class name is a good direction. This name could have stayed as it was, short "PcmEvent". Or could even be simplified further into something like "Kind" / "EventType". It's just a suggestion to consider.

Reply:
I agree you. I think `EventType` is good. I will fix it and request review again.
Thank you.